python/nano/example/pytorch/quantization/inc/resnet18_cifar.py

```diff
@@ -158,4 +161,52 @@ def configure_optimizers(self):
 )
 
 trainer.fit(model, cifar10_dm)
-trainer.test(model, datamodule=cifar10_dm)
+
+fp32_model = model
+# FP32 testing
+start = time()
+outputs = trainer.test(fp32_model, datamodule=cifar10_dm)
+time1 = time() - start
+log_dict = outputs[0]
+print("FP32 model size: %.3f MB" % fp32_model.model_size)
+print("Throughput: %d s" % time1)
+print("Accuracy: top1=%.2f%%" % (log_dict['test_acc'] * 100))
+
+# Save FP32 checkpoint
+trainer.save_checkpoint('ResNet18_on_CIFAR_FP32_CKPT.pth')
+
+# Run post-training quantization
+quantized = trainer.quantize(fp32_model, calib_dataloader=cifar10_dm.train_dataloader(),
+                             val_dataloader=cifar10_dm.test_dataloader(),
+                             metric=Accuracy(num_classes=10),
+                             # recommend 'pytorch_fx' instead of 'pytorch' to enable automatic
+                             # fusion
+                             framework='pytorch_fx',
+                             approach='static',
+                             tuning_strategy='bayesian',
+                             accuracy_criterion={
+                                 'higher_is_better': True, 'relative': 0.1},
+                             timeout=0,  # Limited time(seconds)
+                             max_trials=10,  # Maximum traverse to get best model
+                             raw_return=True)
+
+# Wrap quantized model as a Pytorch-Lightning Module for testing
+int8_model = copy.deepcopy(fp32_model)
+int8_model.model = quantized
```

Comment:
These two steps are very confusing; why do we need to do these?

Reply:
INC outputs a `GraphModule` instead of `Lightning` object. Here I tried to convert it back to `Lightning`. I replaced these lines with:  
```python
int8_model = trainer.compile(int8_model, loss=criterion, metrics=[accuracy])
```